src/lib/checkbox/checkbox.html

```diff
@@ -1,15 +1,22 @@
 <template>
-  <div class="forge-checkbox-wrapper forge-form-field" part="root">
-    <div class="mdc-checkbox" part="container">
-      <slot name="input"></slot>
-      <div class="mdc-checkbox__background" part="background">
-        <svg class="mdc-checkbox__checkmark" viewBox="0 0 24 24" part="checkmark-svg">
-          <path class="mdc-checkbox__checkmark-path" fill="none" d="M1.73,12.91 8.1,19.28 22.79,4.59" part="checkmark-path"></path>
+  <div class="forge-checkbox" part="checkbox">
+    <div id="container" class="container" part="input-container">
+      <slot name="input">
+        <input id="input" class="input" type="checkbox" part="input" />
+      </slot>
+      <div class="background" part="background">
+        <svg class="icon icon--checked" aria-hidden="true" viewBox="0 0 24 24" part="checkmark">
+          <path class="path path--checkmark" d="M1.73,12.91 8.1,19.28 22.79,4.59" part="checkmark-path"></path>
```

Comment:
I'm torn on whether the `<path>` and `<line>` elements should be exposed as `part` elements in this template, thoughts?

My only hesitation is that if we expose everything then it's part of the public API of the element. Are all of these elements we expect developers to need access to or can they be hidden implementation details?

Reply:
I kept the `part` attributes here just because the old checkbox had them, but I don't think it makes a lot of sense. They aren't even targeted by an of the internal styles.